app/src/test/kotlin/com/intive/picover/common/TestUtils.kt

```diff
@@ -3,6 +3,7 @@ package com.intive.picover.common
 import io.mockk.MockKStubScope
 
 fun <T> mockkAnswer(block: MockKStubScope<T, T>.() -> Unit) =
+	@Suppress("ktlint:standard:function-naming")
```

Comment:
@michal-kucznerowicz It's seems to be a ktlint bug, would be nice to create an issue for them